client/src/pages/Home/Home.js

```diff
@@ -10,6 +11,7 @@ class Home extends PureComponent {
     return (
       <div className="Home container">
         <SampleContainer />
+        <ScoreContainer />
```

Comment:
`ScoreContainer` will not be called from this page since `Home` refers to our **Landing Page**. Please remove this line.